.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.10.0
+      - image: cimg/node:12.20.2
```

Comment:
should we update the "engines" blocks in the various components to be >=12.0 now?